resources/views/platform/components/game/heading.blade.php

```diff
@@ -0,0 +1,19 @@
+@props([
+    'consoleName' => 'Unknown Console',
+    'gameTitle' => 'Unknown Game',
+    'iconUrl' => asset("assets/images/system/unknown.png")
+])
+
+<?php
+// TODO: Migrate renderGameTitle to a Blade component.
+$renderedTitle = renderGameTitle(e($gameTitle));
+?>
+
+<h1 class="text-h3">
+    <span class="block mb-1">{{ $renderedTitle }}</span>
```

Comment:
This doesn't work for decorated titles.

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/ace55a58-e849-46a3-99c6-a548206dee0a)

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/b33f9d20-b48f-4c24-8d77-6b2fc54f09de)

Reply:
Resolved by https://github.com/RetroAchievements/RAWeb/pull/1566/commits/f35e86ed06be77c29c15493479320ba9fc098833.